### ISSUE TITLE
Bluetooth connection errors

### ISSUE THREAD

**Issue by TheAirBlow** [ev3dev/ev3dev#1519]
**What are you trying to do?**
Connect a EV3 with `MicroPython` (based on ev3dev) firmware (behavior same on latest original `ev3dev` firmware!) to my Linux laptop by using Bluetooth

**What did you expect to happen**
Simply connect

**What actually happened**
It disconnects shortly after, reconnection on Linux laptop results in the same thing.

Connection on EV3 connects and then shows `GDBus.Error:org.bluex.Error.Failed: Protocol not available`

**Desktop (please complete the following information):**
 - OS and version: Arch Linux (5.15.2-arch1-1)
 - Connection Type: Bluetooth
 - Software: **doesn't matter**

**Robot (please complete the following information):**
 - Device: EV3
 - Versions: **latest MicroPython firmware, version unavailable as it's impossible to connect**

**Comment by dlech** [comment]
Can you please give the exact steps to reproduce the problem? FYI, the *Connect* button on the Bluetooth screen on the EV3 is only for connecting keyboards and game controllers. It doesn't work for anything else, like connecting to a computer.

If you are using Pybricks MicroPython, please see https://pybricks.com/ev3-micropython/messaging.html for instructions.

**Comment by TheAirBlow** [comment]
1) Enable Bluetooth and visibility
2) Connect **a linux-running laptop** using BlueDevil or BlueMan to EV3

BlueDevil will say that `setup of ev3dev failed`, and trying to connect again will show `br-connection-profile-unavailable`
BlueMan does not show any errors at all

**Comment by dlech** [comment]
Yes, this is expected. The EV3 doesn't provide any generic services like a keyboard or Bluetooth speaker, so connecting like this doesn't do anything useful. If you want a network connection, try https://www.ev3dev.org/docs/tutorials/connecting-to-the-internet-via-bluetooth/.

**Comment by TheAirBlow** [comment]
I'm trying to connect EV3 to my linux machine (PC)

вт, 16 нояб. 2021 г., 21:57 David Lechner ***@***.***>:

**Comment by geoffpaulsen** [comment]
I too can't "see" the ev3dev device when trying to pair via bluetooth.
Enabling tethering on chrombook or on EV3dev didn't seem to help.

It does appear to work as normal from a Macbook.  This is my 2nd chromebook I've tried.

EV3 is running Bickman 0.10.3,
Chrombook is a amd64 chromebook.
